index.html

```diff
@@ -1565,45 +1565,40 @@ <h3>Service properties</h3>
     </section>
 
     <section>
-      <h2>DID Subject</h2>
+      <h2>Identification</h2>
```

Comment:
```suggestion
      <h2>Identifiers</h2>
```

"Identification" seems a bit too close to the "identity" side of "identity" vs "identifiers"